File: liqp-core/src/main/java/liqp/tags/Break.java
package liqp.tags;

import liqp.tag.LTag;
import liqp.node.LNode;
import liqp.context.LContext;

public class Break extends LTag {
  @Override
  public Object render(LContext context, LNode... nodes) {
    return BREAK;
  }
}

File: liqp-core/src/main/kotlin/liqp/tags/Break.kt
package liqp.tags

import liqp.ControlResult
import liqp.context.LContext
import liqp.node.LNode
import liqp.tag.LTag

class Break : LTag() {
  override fun render(context: LContext, vararg nodes: LNode): Any? {
    return ControlResult.BREAK
  }
}
